DNN Platform/Providers/FolderProviders/AzureFolderProvider/AzureFolderProvider.cs

```diff
@@ -270,6 +270,21 @@ protected override void MoveFolderInternal(FolderMappingInfo folderMapping, stri
             this.ClearCache(folderMapping.FolderMappingID);
         }
 
+        public override void UpdateFile(IFolderInfo folder, string fileName, Stream content)
+        {
+            // Azure is case sensitive.  If you update dnninternals.pdf with DNNINTERNALS.pdf, to DNN it's the same
+            // so it just re-uploads it, causing both files to exist in Azure.
+            IFileInfo originalFile = FileManager.Instance.GetFile(folder, fileName);

```

Comment:
```suggestion

        /// <summary>
        /// Updates a file in Azure folder provider.
        /// </summary>
        /// <remarks>
        /// Azure is case sensitive.  If you update dnninternals.pdf with DNNINTERNALS.pdf, to DNN it's the same
        /// so it just re-uploads it, causing both files to exist in Azure. This azure specific method deletes the
        /// old existing duplicate that only differs in case as part of the update.
        /// </remarks>
        public override void UpdateFile(IFolderInfo folder, string fileName, Stream content)
        {
            IFileInfo originalFile = FileManager.Instance.GetFile(folder, fileName);

```